app/models/manageiq/providers/vmware/infra_manager/vm.rb

```diff
@@ -5,6 +5,10 @@ class ManageIQ::Providers::Vmware::InfraManager::Vm < ManageIQ::Providers::Infra
   include_concern 'RemoteConsole'
   include_concern 'Reconfigure'
 
+  supports :clone do
+    unsupported_reason_add(:clone, 'Clone operation is not supported') if blank? || orphaned? || archived?
```

Comment:
1) I added `_()` around the message
2) I believe `blank?` it just additional check if the object does not `nil` or does not `empty` I do not sure if this check relevant at all, but I can imagine the situation that we run clone command on the object that already does not exist(via REST API or maybe because some delay between UI and database)